library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java

```diff
@@ -267,11 +264,7 @@ public void onMapClick(LatLng latLng) {
   }
 
   @Override public void setMyLocationEnabled(boolean enabled) {
-    if (myLocationEnabled != enabled) {
-      if (RuntimePermissionUtils.checkLocationPermissions(getActivity(), this)) {
```

Comment:
Setting the `myLocationEnabled` flag here on `NativeGoogleMapFragment` doesn't require location permissions. If anything I think this check should live on within `onLocationPermissionsGranted` here https://github.com/forrestbice/AirMapView/blob/30b6f1e28836e6c9e737cd41548b64b761b23035/library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java#L270 but since the caller is responsible for calling `onLocationPermissionsGranted` it seemed fine to offload the responsibility of the permission check as well. Note: The `//noinspection MissingPermission` lint flag was already present within `onLocationPermissionsGranted`